bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1408,12 +1435,12 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
          lnd_tuning_mode="clm5_1_GSWP3v1" use_init_interp=".true."
 >lnd/clm2/initdata_esmf/ctsm5.2/clmi.I2000Clm50BgcCrop.2011-01-01.1.9x2.5_gx1v7_gl4_simyr2000_c240223.nc
 </finidat>
-<!-- NB: This pSASU finidat is from an f09 1850 (rather than f19 2000) spin-up -->
+<!-- NB: This pSASU finidat is from an f09 2000 spin-up -->
```

Comment:
Probably should just remove this line now.

Reply:
Removing. Unless we still want to point out that the finidat is f09 for this f19 case.